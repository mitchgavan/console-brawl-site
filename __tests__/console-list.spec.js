import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ConsoleList from '../components/console-list/console-list';

test('It renders correctly', () => {
  const tree = shallow(
    <ConsoleList />,
  );
  expect(toJson(tree)).toMatchSnapshot();
});
