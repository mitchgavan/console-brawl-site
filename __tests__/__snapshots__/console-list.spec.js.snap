// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<withHandlers(Component)
  active={[Function]}
  toggledOn="All"
/>
`;
